tools/online_editor/src/worker/lsp_worker.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import {
+  createConnection,
+  BrowserMessageReader,
+  BrowserMessageWriter,
+} from "vscode-languageserver/browser";
+import {
+  ExecuteCommandParams,
+  InitializeParams,
+  InitializeResult,
+} from "vscode-languageserver";
+
+import slint_init, * as slint_lsp from "@lsp/slint_lsp_wasm.js";
+
+console.log("*** LSP WEBWORKER STARTUP INITIATED! ***");
+
+slint_init().then((_) => {
+  const reader = new BrowserMessageReader(self);
+  const writer = new BrowserMessageWriter(self);
+
+  let the_lsp: slint_lsp.SlintServer;
+
+  const connection = createConnection(reader, writer);
+
+  function send_notification(method: string, params: unknown): boolean {
+    connection.sendNotification(method, params);
+    return true;
+  }
+
+  async function load_file(path: string): Promise<Uint8Array> {
+    return await connection.sendRequest("slint/load_file", path);
```

Comment:
i guess this should be a comment that this is a TODO for now,  as the client currently don't understand that.

Reply:
Todo added.